walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -19,40 +19,60 @@
 import java.util.List;
 
 public final class FindMeetingQuery {
+
+  private static final int MINUTES_IN_ONE_DAY = 1440;
+
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     long duration = request.getDuration();
-    int[] minutes = new int[1441];
+    int[] minutes = new int[MINUTES_IN_ONE_DAY];
```

Comment:
From just reading the code, the array `minutes` filled with 0, 1, 2 doesn't tell the reader what's going on. Consider renaming it to `availableMinutes`, `takenMinutes` or `takenTimeSlots` or whatever comes to your mind. Also please leave the comment describing the values 0, 1, 2 near the definition.

In the real world, I guess I would fill this array with enum values:
enum {
  IS_FREE,
  IS_TAKEN,
  IS_TAKEN_BY_OPTIONAL,
}
In this case it's clear what are the meanings of this magical 0, 1, 2. But for this project I thinks it's overhead, so please just leave a comment.

Another approach would have been to solve 2 different problems: first including optional attendees and second -- without them. In that case you only need boolean values for availability. A bit less optimal, but same in terms of big-O. Again, I think for the starter project it's enough to just leave the comment :)

Reply:
Thanks, I will change the name of variable and add comments. I will use enum for future work.